--- setup-2.4.6/etc/profile.d/tmp-dir.csh~	Sun Jun 13 22:15:53 2004
+++ setup-2.4.6/etc/profile.d/tmp-dir.csh	Sun Jun 13 22:15:53 2004
@@ -1,4 +1,4 @@
-if ( ! -d ~/tmp ) then
+if ( `/bin/id -u` >= 500 && ! -d ~/tmp ) then
 	mkdir -p ~/tmp >& /dev/null
-	chmod -f og-rwx ~/tmp
+	chmod -f og-rwx ~/tmp >& /dev/null
 endif
--- setup-2.4.6/etc/profile.d/tmp-dir.sh~	Sun Jun 13 22:15:53 2004
+++ setup-2.4.6/etc/profile.d/tmp-dir.sh	Sun Jun 13 22:15:53 2004
@@ -1,4 +1,4 @@
-if [ ! -d ~/tmp ]; then
+if [ $(/bin/id -u) -ge 500 -a ! -d ~/tmp ]; then
 	mkdir -p ~/tmp > /dev/null 2>&1
-	chmod -f og-rwx ~/tmp
+	chmod -f og-rwx ~/tmp 2>/dev/null
 fi
