diff -urN setup-2.4.5.orig/etc/fstab setup-2.4.5/etc/fstab
--- setup-2.4.5.orig/etc/fstab	2004-02-22 09:24:41.000000000 +0100
+++ setup-2.4.5/etc/fstab	2004-02-22 09:25:20.000000000 +0100
@@ -3,7 +3,7 @@
 
 none			/proc			proc	defaults,gid=17	0 0
 none			/dev/pts		devpts	gid=5,mode=620	0 0
-#none			/dev/shm		shmfs	defaults	0 0
+#none			/dev/shm		tmpfs	defaults	0 0
 #none			/dev			devfs	defaults	0 0
 
 /dev/fd0		/media/floppy		auto	user,noauto	0 0
