--- ./etc/services.org	2005-05-26 20:27:14.000000000 +0200
+++ ./etc/services	2007-07-19 01:21:50.000000000 +0200
@@ -12776,6 +12776,8 @@
 pago-services1  30001/udp   # Pago Services 1
 pago-services2  30002/tcp   # Pago Services 2
 pago-services2  30002/udp   # Pago Services 2
+#
+csync2		30865/tcp   # Cluster sync
 #                           Balduin Mueller-Platz 
 #                           <balduin.mueller-platz@pago.de> March 2002
 #               30003-31415 Unassigned
