Index: profile
===================================================================
RCS file: /cvsroot/setup/etc/profile,v
retrieving revision 1.48
diff -u -r1.48 profile
--- ./etc/profile	26 May 2005 18:27:14 -0000	1.48
+++ ./etc/profile	30 Apr 2006 11:23:40 -0000
@@ -115,19 +115,24 @@
 # example:
 # cat /etc/env.d/VARIABLE
 # VARIABLE="value"
-for i in /etc/env.d/* ; do
-	NAME=`basename $i`
-	case $NAME in
-		*~ | *.bak | *.old | *.rpmnew | *.rpmsave )
-			# nothing
-			;;
-		* )
-			if [ -r $i ]; then
-				. $i; export $NAME
-			fi
-			;;
-	esac
-done
+if [ -f /etc/profile.env ]; then
+	. /etc/profile.env
+else
+	for i in /etc/env.d/* ; do
+		NAME=${i##*/}
+		case $NAME in
+			*~ | *.bak | *.old | *.rpmnew | *.rpmsave )
+				# nothing
+				;;
+			* )
+				if [ -r $i ]; then
+					. $i; export $NAME
+				fi
+				;;
+		esac
+	done
+	unset NAME
+fi
 
 # Scripts:
 for i in /etc/profile.d/*.sh ; do
