diff -urN setup-2.4.5.orig/etc/profile setup-2.4.5/etc/profile
--- setup-2.4.5.orig/etc/profile	2003-09-26 10:40:01.000000000 +0200
+++ setup-2.4.5/etc/profile	2004-02-22 09:28:21.000000000 +0100
@@ -24,7 +24,8 @@
 
 # home_etc user config-dirs
 #CONFIG_DIR=etc
-#export CONFIG_DIR
+#HOME_ETC=$HOME/$CONFIG_DIR
+#export CONFIG_DIR HOME_ETC
 
 #
 # Setup the environment for various shells
@@ -125,7 +126,6 @@
   zsh)
 	PS1='[%n@%m %~]%(!.#.%\$) '
 	RPS1="%T"
-	export RPS1
 
 	# other HISTFILE, zsh uses diffrent fmt
 	HISTFILE=~/.historyz
