diff -urN setup-2.4.6.orig/etc/fstab setup-2.4.6/etc/fstab
--- setup-2.4.6.orig/etc/fstab	2004-04-04 21:19:54.000000000 +0200
+++ setup-2.4.6/etc/fstab	2004-04-28 13:07:56.714742808 +0200
@@ -1,10 +1,16 @@
-#/dev/hda1		swap			swap	defaults	0 0
-#/dev/hda2		/			ext2	defaults	1 1
+#/dev/hda1	swap			swap	defaults		0 0
+#/dev/hda2	/			ext2	defaults		1 1
 
-none			/proc			proc	defaults,gid=17	0 0
-none			/dev/pts		devpts	gid=5,mode=620	0 0
-#none			/dev/shm		tmpfs	defaults	0 0
-#none			/dev			devfs	defaults	0 0
+none		/proc			proc	defaults,noauto,gid=17	0 0
+none		/sys			sysfs	defaults,noauto		0 0
+none		/proc/bus/usb		usbfs	defaults,noauto		0 0
+none		/dev			devfs	defaults,noauto		0 0
+none		/dev/pts		devpts	gid=5,mode=620		0 0
+#none		/dev/shm		tmpfs	defaults		0 0
 
-/dev/fd0		/media/floppy		auto	user,noauto	0 0
-/dev/cdrom		/media/cdrom		auto	users,noauto	0 0
+/dev/fd0	/media/floppy		auto	user,noauto		0 0
+/dev/cdrom	/media/cdrom		auto	user,noauto		0 0
+
+# Examples for mount vfat and ntfs patritions using fsctrl group
+#/dev/hdXX	/mnt/ntfs	ntfs	user,noauto,umask=002,uid=0,gid=28,iocharset=iso8859-2 0 0
+#/dev/hdXX	/mnt/vfat	vfat	user,noauto,umask=002,uid=0,gid=28,iocharset=iso8859-2,codepage=852,exec,showexec 0 0
diff -urN setup-2.4.6.orig/etc/group setup-2.4.6/etc/group
--- setup-2.4.6.orig/etc/group	2004-04-04 21:19:54.000000000 +0200
+++ setup-2.4.6/etc/group	2004-04-28 13:00:14.227051624 +0200
@@ -23,6 +23,7 @@
 audio::23:
 video::24:
 cdwrite::27:
+fsctrl::28:
 console::31:
 dip::40:
 ftp::50:
@@ -30,6 +31,7 @@
 nobody::99:
 stats::123:
 logs::124:stats
+fileshare::138:
 users::1000:
 tpexec::65500:
 nosock::65501:
