Index: etc/profile
===================================================================
RCS file: /cvsroot/setup/etc/profile,v
retrieving revision 1.55
retrieving revision 1.57
diff -u -u -r1.55 -r1.57
--- setup/etc/profile	22 Jan 2011 20:08:54 -0000	1.55
+++ setup/etc/profile	24 Feb 2011 23:07:56 -0000	1.57
@@ -128,7 +128,7 @@
 				;;
 			* )
 				if [ -r "$i" ]; then
-					. "$i"; export "$NAME"
+					. "$i"; eval [ '"${'"$NAME"'+X}"' = 'X' ] && export $NAME
 				fi
 				;;
 		esac
