--- setup-2.4.4/etc/services.orig	Thu Sep 25 19:47:41 2003
+++ setup-2.4.4/etc/services	Thu Sep 25 20:01:00 2003
@@ -9721,7 +9721,7 @@
 pcanywherestat	5632/udp		# pcANYWHEREstat
 #			   Jon Rosarky <JRosarky@symantec.com>
 #		5633-5672  Unassigned
-jms		5673/tcp		# JACL Message Server
+jms		5673/tcp irrd		# IRRd routing registry server [jms - JACL Message Server]
 jms		5673/udp		# JACL Message Server
 #			   Stuart Allen <stuart@jacl.animats.net> February 2002
 hyperscsi-port	5674/tcp		# HyperSCSI Port
@@ -10490,7 +10490,7 @@
 cddbp-alt	8880/udp 		# CDDBP
 #			    Steve Scherf <steve@moonsoft.com>
 #		8881-8887   Unassigned
-ddi-tcp-1	8888/tcp 		# NewsEDGE server TCP (TCP 1)
+ddi-tcp-1	8888/tcp irr_rpsl_submit # NewsEDGE server TCP (TCP 1)
 ddi-udp-1	8888/udp 		# NewsEDGE server UDP (UDP 1)
 ddi-tcp-2	8889/tcp 		# Desktop Data TCP 1
 ddi-udp-2	8889/udp 		# NewsEDGE server broadcast
